keyboards/spleeb/readme.md

```diff
@@ -0,0 +1,110 @@
+# spleeb
+
+![spleeb](https://i.imgur.com/2rmZa6M.jpg)
```

Comment:
This uses a smaller, but still high res version of the image that is more comaptible with github's markdown rendering. 
```suggestion
![spleeb](https://i.imgur.com/2rmZa6Mh.jpg)
```

Reply:
Thank you! I addressed this in the fixup commit.